cmd/kubeadm/app/preflight/checks.go

```diff
@@ -946,28 +946,22 @@ func RunJoinNodeChecks(execer utilsexec.Interface, cfg *kubeadmapi.JoinConfigura
 		checks = append(checks, FileAvailableCheck{Path: cfg.CACertPath})
 	}
 
-	addIPv6Checks := false
 	if cfg.Discovery.BootstrapToken != nil {
 		ipstr, _, err := net.SplitHostPort(cfg.Discovery.BootstrapToken.APIServerEndpoint)
-		if err == nil {
-			checks = append(checks,
-				HTTPProxyCheck{Proto: "https", Host: ipstr},
-			)
-			if !addIPv6Checks {
-				if ip := net.ParseIP(ipstr); ip != nil {
-					if ip.To4() == nil && ip.To16() != nil {
-						addIPv6Checks = true
-					}
-				}
+		if err != nil {
+			fmt.Printf("[preflight] WARNING: Couldn't split host and port: %v\n", err)
```

Comment:
Should this warning be a little more specific? When I think about this message as a user it seems a little confusing. Maybe some additional context would help, something like "WARNING: Could not split host and port of the APIServerEndpoint"? Not tied to that specific language. Any words that provide an additional hint beyond some host/port that fails to parse properly.

Reply:
so it was originally more detailed, but i requested the simplification due to the fact that `SplitHostPort` prints the failed `host:port` string.
but adding the word `APIServerEndpoint` won't hurt.

@chuckha please confirm if you'd prefer to still have the word.

Reply:
gotcha, I missed the past discussion, my fault. I'm good with leaving this message as-is since you already asked for simplification. The errors from `net.SplitHostPort` [look pretty good][https://golang.org/src/net/ipsock.go?s=4746:4812#L145] and should be more than enough information to get the user on their way. If it becomes confusing, users will tell us and we can fix it later.